src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/ErsedCalculator.kt

```diff
@@ -0,0 +1,137 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.service
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Configuration
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.enumerations.CalculationRule
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.enumerations.ReleaseDateType
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.AdjustmentDuration
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.CalculableSentence
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.Offender
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.ReleaseDateCalculationBreakdown
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.SentenceCalculation
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.util.isAfterOrEqualTo
+import java.time.temporal.ChronoUnit
+import kotlin.math.ceil
+import kotlin.math.max
+
+@Service
+class ErsedCalculator(val ersedConfiguration: ErsedConfiguration) {
```

Comment:
I can't see where this is used? Its mostly talking about HDCED too.

Reply:
Yup - we'll remove this one as this was a desired change but we won't do it now due to time constraints.